### ISSUE TITLE
VIAC PDF import

### ISSUE THREAD

**Issue by lucacame** [portfolio-performance/portfolio#1664]
The PDF import from VIAC only works if the PDF is in German, but raises an error with other languages.
Could it be possible to add the English translation? I can provide the PDFs with the right wording.

**Comment by inv-trad** [comment]
@lucacame 
Hi Luca,

just post an anonymised PDF extract here and I'll have a look.

Go in PP to menu “File” -> “Import…” -> “Debug: Extract text from PDF".
Please use the "Insert code" button <> to not have it formatted.

Thanks!

**Comment by lucacame** [comment]
Thanks @inv-trad ! Here the three different types of documents, in English

Exchange settlement:

```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 03.07.2020
Exchange Settlement - Buy
We have executed the following order for you:
Order: Buy
0.114 Qty UBS ETF MSCI EMU SRI
ISIN: LU0629460675
Price: EUR 93.09
Amount EUR 10.58
Exchange rate CHF/EUR 1.0672 CHF 11.29
Stamp duty CHF 0.01
Charged amount: Value date 03.07.2020 CHF 11.30
S. E. & O.
Kind regards
Terzo Pension Foundation
Display without signature
```

Interest settlement:
```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 01.07.2020
Interest
On 30.06.2020 we have credited your account:
Interest rate: 0.10%
Interest period: June
Interest credit: CHF 0.05
Charged amount: CHF 0.05
S. E. & O.
Kind regards
Terzo Pension Foundation
Display without signature
```

Fees settlement:

```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 01.07.2020
Commission
On 30.06.2020 we have debited your account:
VIAC administration fee: 0.52% p.a.
Average pension assets: CHF 10'000.00
Less 3a account balance (liquidity): CHF 1'000.00
Effective calculation basis: Month June CHF 9'000.00
Effective VIAC administration fee: 0.47% p.a. CHF -42.30
Charged amount: Value date 30.06.2020 CHF -42.30
S. E. & O.
Note: Whenever possible, our foundation bears the product costs (index funds). In the case of ETFs, product costs are charged within the fund and 
accrue in favor of the fund provider. Product costs accrue in addition to the administration fee and amount to 0.09 % per year for the currently 
selected strategy. VIAC does not receive any kind of reimbursements (retrocessions).
Kind regards
Terzo Pension Foundation
Display without signature
```

**Comment by inv-trad** [comment]
@lucacame 
Thanks. Buy, Interest and Fee Transactions are implemented, still missing Deposit, Sell and Dividend Transactions. 
If you happen to have examples here, too, feel free to send them as well.

**Comment by lucacame** [comment]
@inv-trad thanks for the quick implementation!
Here the other transaction documents.
Sell:

```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 05.02.2020
Exchange Settlement - Sell
We have executed the following order for you:
Order: Sell
0.035 Qty CSIF SMI
ISIN: CH0033782431
Price: CHF 1'379.01
Amount CHF 48.24
Charged amount: Value date 05.02.2020 CHF 48.24
S. E. & O.
Note: Whenever possible, our foundation bears the product costs (index funds). In the case of ETFs, product costs are charged within the fund and 
accrue in favor of the fund provider. Product costs accrue in addition to the administration fee and amount to 0.09 % per year for the currently 
selected strategy. VIAC does not receive any kind of reimbursements (retrocessions).
Kind regards
Terzo Pension Foundation
Display without signature
```

Dividend:

```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 27.02.2020
Dividend Payment
We have booked the following dividend payment for you:
Type of dividend: Ordinary dividend
2.142 Qty iShares US Property Yield
ISIN: IE00B1FZSF77
Dividend payment: USD 0.25
Amount USD 0.54
Exchange rate CHF/USD 0.9645 CHF 0.52
Amount credited: Value date 27.02.2020 CHF 0.52
S. E. & O.
Note: Whenever possible, our foundation bears the product costs (index funds). In the case of ETFs, product costs are charged within the fund and 
accrue in favor of the fund provider. Product costs accrue in addition to the administration fee and amount to 0.09 % per year for the currently 
selected strategy. VIAC does not receive any kind of reimbursements (retrocessions).
Kind regards
Terzo Pension Foundation
Display without signature
```

Deposit:

```
PDF author: 'VIAC'
PDFBox Version: 1.8.16
-----------------------------------------
Terzo Vorsorgestiftung der WIR Bank
Auberg 1
4002 Basel
Internet www.viac.ch
Email info@viac.ch
Phone 0800 80 40 40
Contract x.xxx.xxx.xxx Mr John Doe
Portfolio x.xxx.xxx.xxx.xx 
Bahnhofstrasse 1
8001 Zürich
Basel, 17.12.2019
Deposit 3a
Credit from 17.12.2019:
Incoming payment from: Deposit BESR
Reference number: xx xxxxx xxxxx xxxxx xxxxx xxxxx 
Incoming payment: Amount CHF 1'000.00
Credit: Value date 17.12.2019 CHF 1'000.00
S. E. & O.
Note: Whenever possible, our foundation bears the product costs (index funds). In the case of ETFs, product costs are charged within the fund and 
accrue in favor of the fund provider. Product costs accrue in addition to the administration fee and amount to 0.09 % per year for the currently 
selected strategy. VIAC does not receive any kind of reimbursements (retrocessions).
Kind regards
Terzo Pension Foundation
Display without signature
```

**Comment by inv-trad** [comment]
Thanks. Is done as well.

**Comment by Nirus2000** [comment]
This issue is closed with #1676
